apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -160,6 +162,9 @@ html, body{
         padding: 5px 10px 5px 53% !important;
     }
 }
+.text-right{
```

Comment:
css is already written in bootstrap.css @prateek-giddh

Reply:
Done